main.go

```diff
@@ -110,17 +110,25 @@ func homeDir() string {
 }
 
 func loadConfig(kubeconfig string) (*rest.Config, error) {
-	if c, err := rest.InClusterConfig(); err == nil {
-		return c, nil
+	if len(kubeconfig) > 0 {
+		return clientcmd.BuildConfigFromFlags("", kubeconfig)
 	}
-	home := homeDir()
-	if kubeconfig == "" && home != "" {
-		kubeconfig = filepath.Join(home, DefaultKubeConfigFilePath)
+
+	kubeconfigPath := os.Getenv(clientcmd.RecommendedConfigPathEnvVar)
+	if len(kubeconfigPath) > 0 {
+		envVarFiles := filepath.SplitList(kubeconfigPath)
+		for _, f := range envVarFiles {
+			if _, err := os.Stat(f); err == nil {
+				return clientcmd.BuildConfigFromFlags("", f)
+			}
+		}
 	}
-	_, err := os.Stat(kubeconfig)
-	if err != nil {
-		return nil, err
+
+	if c, err := rest.InClusterConfig(); err == nil {
```

Comment:
Hmm? So the priority would be:
1. ENV/RecommendedDir
2. In-cluster
3. RecommendedFileName 

Is this intended? I found it's weird that in-cluster is behind recommended dir but before the recommended file.

Also, do we still need `DefaultKubeConfigFilePath`?

Reply:
Oh... The priority of kubeconfig would be:

1.  Command args.  We could specify the --kubeconfig.
2. ENV.  We read from RecommendedConfigPathEnvVar.
3. In-cluster.
4. Recommended. We got by RecommendedHomeDir and RecommendedFileName.

Reply:
That makes sense. Can you add this order to the commit message? Otherwise LGTM.

Reply:
OK.